### ISSUE TITLE
Feature Request: Add option to read MPRIS2 information

### ISSUE THREAD

**Issue by freijon** [masmu/pulseaudio-dlna#204]
It would be nice if there was an option to detect when the song has changed that is playing with any music player and update the text displayed with the Chromecast. MPRIS2 provides this information. Maybe even change the displayed image to the album cover of the current song.

**Comment by masmu** [comment]
I've investigated that feature some time ago. It is not possible with DLNA, because there is no call in the DLNA specification which just changes the cover art. It is just possible to set the playlist and every time you do this the device starts the track again.
For Chromecasts it is not possible using the default build-in media player. He is missing the same call. But you can write your own little media player by creating a custom Chromecast app.

To sum it up: There would be a lot new code just for getting the MPRIS2 information and I had to write an own media player. And in the end this is just usable with Chromecasts. Thats why I put this to the end of my list. I really would like to have that feature myself, but it felt like this is too much just for one device family.
